grapher/mapCharts/AnnotationGenerator.ts

```diff
@@ -664,14 +587,24 @@ function getLabelInfo(
                 }
             }
         }
-        polylabelCache.push({
+        internalInfo.push({
             position: pos.slice(0, 2),
             points: regionPoints,
             id: id,
         })
     }
-    annotationsCache.get(projection).set("polylabels", polylabelCache)
-    annotationsCache.get(projection).set("allPoints", allPoints)
+    annotationsCache.get(projection)!.regions = regions
+    annotationsCache.get(projection)!.internalInfo = internalInfo
+    annotationsCache.get(projection)!.allPoints = allPoints
+}
+
+function getRatio(coordinates: any): number {
+    const bounds = geoPath().bounds({ type: "Polygon", coordinates })
+    const dx = bounds[1][0] - bounds[0][0]
+    const dy = bounds[1][1] - bounds[0][1]
+    const ratio = dx / (dy || 1)
+    const A = 12
+    return Math.min(Math.max(ratio, 1 / A), A)
 }
```

Comment:
It seems like `getRatio` is only relevant in `polylabelStretched` and all the parameters it needs are already passed into `polylabelStretched` – is it maybe possible to use it inside `polylabelStretched` and not have to call it each time separately?